### PR TITLE
f-header@4.11.2 - Fixed typo in Italian header

### PR DESCRIPTION
**Pull request by 1itvinka**
### Fixed
- Fixed typo in Italian header

### CODE REVIEW

File: packages/components/organisms/f-header/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-header",
   "description": "Fozzie Header – Globalised Header Component",
-  "version": "4.11.1",
+  "version": "4.12.0",
```
Comment (1itvinka) [inline packages/components/organisms/f-header/package.json]: is it a correct version upd for a typo fix? or should it be `4.11.2` instead?

File: packages/components/organisms/f-header/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
 
+v4.12.0
+------------------------------
+*March 16, 2021*
+
+### Fixed
+- Fixed typo in Italian header
+
 
 Latest (add to next release)
```
Comment (1itvinka) [inline packages/components/organisms/f-header/CHANGELOG.md]: is it okay to add my changelog comment above this one? Or should I edit the previous one ("Latest (add to next release)") somehow?

Comment (kevinrodrigues) [reply packages/components/organisms/f-header/CHANGELOG.md]: You should combine the changelog from here with your versioned changes.

File: packages/components/organisms/f-header/src/tenants/it-IT.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ export default {
             gtm: 'click_account_your_orders'
         },
         accountCredit: {
-            text: 'Credito dell',
+            text: "Credito dell'account",
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-header/src/tenants/it-IT.js:21]: You should be able to escape the `'` with a backslash so you can keep the single quotes.

File: packages/components/organisms/f-header/package.json (hunk 1 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-header/package.json]: Think `4.11.2` would make sense here.

File: packages/components/organisms/f-header/src/tenants/it-IT.js (hunk 3 above)
Comment (1itvinka) [reply packages/components/organisms/f-header/src/tenants/it-IT.js:21]: Do you have any policies or guidelines on this? In f-footer we use the format I used here. And as for me, it's more readable anyway.

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**